src/Benchmarks.ServerJob/project.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "Repository": "1.0.0-*",
-    "Newtonsoft.Json": "8.0.3"
```

Comment:
I don't see a reason why we'd want this to be less than our "framework minimum" even though it hypothetically could be.